README.md

```diff
@@ -471,6 +471,25 @@ In the meantime, the plugin may be installed via the [Obsidian BRAT plugin](http
 
 You can follow the same procedure to update the plugin.
 
+### Building from Source
+
+After installing the plugin, you can use `npm` to build new `main.js` and `styles.css` files from source.
+
+-   Open the source directory in your terminal and run `npm install`.
+-   Running `npm run build` will build the `main.js` and `styles.css` files inside the source directory.
+-   Navigate to your `fantasy-calendar` folder in your vault's plugins folder and replace the `main.js` and `styles.css` files with the newly generated ones.
+-   Reload Obsidian to use the new files.
```

Comment:
Should mention the Obsidian hot reload plugin here. 

https://github.com/pjeby/hot-reload

Reply:
Done! I didn't know about that plugin. Very handy!